client/packages/invoices/src/InboundShipment/api/hooks/line/useZeroInboundLinesQuantity.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  useTableStore,
+  useTranslation,
+  useDeleteConfirmation,
+} from '@openmsupply-client/common';
+import { useIsInboundDisabled } from '../utils/useIsInboundDisabled';
+import { useInboundRows } from './useInboundRows';
+import { useZeroInboundLineQuantity } from './useZeroInboundLineQuantity';
+
+export const useZeroInboundLinesQuantity = (): (() => void) => {
```

Comment:
This hook feels out of place here because it also contains the UI part and the very specific part of reading the data from the table. You should either create a hook that is UI independent, e.g. where you pass in the table data as a parameter, or move this file closer to where it is used (or put the hook into Toolbar.tsx?).

Might be wrong but only saw hooks that directly map to an api in this hooks directory... For example, found it confusing to have a useZeroInboundLineQuantity and a useZeroInboundLinesQuantity hook where the former actually zeros multiple lines and the second does UI stuff..

Reply:
Was just keeping things consistent since that's how it's done in the rest of the project~